Doc/library/doctest.rst

```diff
@@ -300,33 +301,49 @@ their contained methods and nested classes.
 How are Docstring Examples Recognized?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The :mod:`doctest` module treats any line beginning with ``>>>`` as an 
+example to be tested. 
+Following lines which begin with ``...`` continue the example. 
+Subsequent non-blank lines, if any, form an expected output string. 
+A blank (all-whitespace) line, or a line with ``>>>`` (beginning the 
+next example), ends the expected output string.
+
 In most cases a copy-and-paste of an interactive console session works fine,
 but doctest isn't trying to do an exact emulation of any specific Python shell.
 
 ::
 
    >>> # comments are ignored
-   >>> x = 12
-   >>> x
-   12
-   >>> if x == 13:
-   ...     print("yes")
+   ... 
+   >>> import math
+   >>> x = factorial(10); math.ceil(math.log10(x))
+   7
+   >>> if x == factorial(9)*10:
+   ...     print("same:\n{0}".format(x))
    ... else:
-   ...     print("no")
-   ...     print("NO")
-   ...     print("NO!!!")
+   ...     print("differ:\n{0}\n{1}".format(x, factorial(9)*10))
```

Comment:
I'm open to a better example. 
I was trying to demonstrate both a compound statement and a multi-line output string, as well as being clear about setting state in one statement and testing it in the next. What I came up with is pretty contrived. 
Maybe have the test be a while loop?  
Frankly, I didn't find the `if… print("yes")… print("NO!!!")` example all that impressive. It seemed even more contrived, and silly rather than funny. Maybe others appreciate it more.